src/config/commandOptions.js

```diff
@@ -46,6 +46,9 @@ export default {
   noPrependStageInUrl: {
     usage: "Don't prepend http routes with the stage.",
   },
+  noStripTrailingSlashInUrl: {
+    usage: "Don't strip trailing slash from http routes.",
```

Comment:
could you add `type` of the command option as well? It was introduced to all other options in the meantime